bluebottle/activities/tasks.py

```diff
@@ -130,17 +131,17 @@ def recommend():
     ignore_result=True
 )
 def do_good_hours_reminder():
-
     for tenant in Client.objects.all():
         with LocalTenant(tenant, clear_tenant=True):
             settings = MemberPlatformSettings.objects.get()
             if settings.do_good_hours:
-                q1 = date.today().replace(month=1, day=1)
-                q2 = date.today().replace(month=4, day=1)
-                q3 = date.today().replace(month=7, day=1)
-                q4 = date.today().replace(month=10, day=1)
-                notification = None
+                offset = settings.fiscal_month_offset
                 today = date.today()
+                q1 = (date.today().replace(month=1, day=1) + relativedelta(months=offset)).replace(today.year)
```

Comment:
I find this really hard to read. Why do `date.today().replace(month=1, day=1)` when `date(2000, 1, 1)` would also work.
I also find using `date.replace(<int>)` really unintuitive. Better use `date.replace(year=<int>)`